src/components/timeline/TrackCustomMarkerGraph.js

```diff
@@ -152,18 +151,17 @@ class TrackCustomMarkerCanvas extends React.PureComponent<CanvasProps> {
 
         const getX = (time) =>
           Math.round((time - rangeStart) * millisecondWidth);
+        // For line graphs, ensure y is at least half the stroke's line width
+        // so that it won't be cut off the bottom edge of the graph.
+        const minY = type === 'bar' ? 0 : deviceLineWidth * 0.5;
```

Comment:
Is there a reason why we remove the `deviceLineHalfWidth` variable and compute it here? I think for performance-wise it might be better to keep that variable at the top instead of computing inside the loop.

Reply:
I moved it here for clarity so that the "* 0.5" part is near the comment that explains why we need it.

Performance wise, I don't think we'll see a difference. It's now inside the loop that loops over the graphs (I would be surprised if we ever had more than 100 graphs), but remains outside of the loops that loop over samples (which are the hot code path).